tests/test_kea.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+import pytest
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerLauncher
+from pytest_container.runtime import OciRuntimeBase
+
+from bci_tester.data import KEA_CONTAINERS
+
+pytest.mark.parametrize("ctr_image", KEA_CONTAINERS)
+
+
+def test_kea_dhcp4(
+    container_runtime: OciRuntimeBase,
+    pytestconfig: pytest.Config,
+    ctr_image: DerivedContainer,
+) -> None:
+    kea_ctr = DerivedContainer(
+        base=ctr_image,
+        # below base image was used for testing as we haven't published kea image yet
+        # base="registry.opensuse.org/devel/bci/tumbleweed/containerfile/opensuse/kea:latest",
+        containerfile="COPY tests/files/kea-dhcp4.conf /etc/kea/kea-dhcp4.conf",
+        custom_entry_point="kea-dhcp4",
+        extra_entrypoint_args=["-c", "/etc/kea/kea-dhcp4.conf"],
+        extra_launch_args=["--network=host", "--privileged"],
+    )
+
+    dhcp_client_ctr = DerivedContainer(
+        base="registry.opensuse.org/opensuse/tumbleweed:latest",
+        containerfile="RUN zypper refresh && zypper -n install dhcp-client && zypper clean --all",
+        custom_entry_point="/bin/sh",
+        extra_launch_args=["--network=host", "--privileged"],
+    )
+
+    with ContainerLauncher.from_pytestconfig(
+        kea_ctr, container_runtime, pytestconfig
+    ) as kea_launcher, ContainerLauncher.from_pytestconfig(
+        dhcp_client_ctr, container_runtime, pytestconfig
+    ) as cli_launcher:
+        kea_launcher.launch_container()
+        cli_launcher.launch_container()
+
+        cli_con = cli_launcher.container_data.connection
+        client_log = cli_con.run_expect([0], "dhclient -v wlp0s20f3").stderr
```

Comment:
I'm pretty certain this will not work universally. At least the github CI cloud workers and openQA workers don't have wifi